### PR TITLE
upgrade dependencies to fix Vulnerabilities issues

### PR DESCRIPTION
**Pull request by dengmingtong**
Issue #, if available:

Description of changes:

Checklist:

- [ yes] Have you successfully deployed to an AWS account with your changes?
- [ yes] Have you successfully deployed to an AWS China account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: addons/addon-base-post-deployment/packages/base-post-deployment/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "@amzn/base-api-services": "workspace:*",
     "@amzn/base-services": "workspace:*",
     "@amzn/base-services-container": "workspace:*",
-    "aws-sdk": "^2.1000.0",
```
Comment (zxkane) [inline addons/addon-base-post-deployment/packages/base-post-deployment/package.json:12]: it looks like a downgrade `aws-sdk` and the latest one is `2.1233.0`, is it expected?

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ importers:
       '@amzn/base-api-services': workspace:*
       '@amzn/base-services': workspace:*
       '@amzn/base-services-container': workspace:*
-      aws-sdk: ^2.1000.0
```
Comment (zxkane) [inline pnpm-lock.yaml:28]: pnmp lock is different with npm-lock or yarn-lock. Make sure the which version of `aws-sdk` is resolved by pnmp.

**Review by zxkane — CHANGES_REQUESTED** [review]

File: addons/addon-base-post-deployment/packages/base-post-deployment/package.json (hunk 1 above)
Comment (dengmingtong) [reply addons/addon-base-post-deployment/packages/base-post-deployment/package.json:12]: Upgrade all to ^2.1233.0 version

File: pnpm-lock.yaml (hunk 2 above)
Comment (dengmingtong) [reply pnpm-lock.yaml:28]: Confirmed

**Review by zxkane — APPROVED** [review]

**Pull request merged**